### ISSUE TITLE
SDK Version Issue in Android

### ISSUE THREAD

**Issue by EjazAmir** [jonasbark/flutter_stripe_payment#323]
"Your app ---- version code --- includes SDK com.stripe:stripe-android, 10.4.6 which facilitates the transmission or collection of personal and sensitive information. Such behavior must comply with Google Play's Prominent Disclosure & Consent Requirements and applicable privacy policy requirements as described in the User Data policy.
Starting from July 11, 2021 midnight (UTC), new app releases containing this SDK version that do not meet the Prominent Disclosure & Consent Requirements per the User Data policy will be rejected. You may consider upgrading to 16.9.0, which the SDK provider has recommended for use instead. Please consult the SDK provider for further information."

Currently, I am using the Version 1.0.11 on Flutter 1.22.6, I can't use the latest version because it's using Dart 2.12 which uses flutter 2.
Any workaround to how to overcome this issue ?

**Comment by ZoraizEjaz** [comment]
> "Your app ---- version code --- includes SDK com.stripe:stripe-android, 10.4.6 which facilitates the transmission or collection of personal and sensitive information. Such behavior must comply with Google Play's Prominent Disclosure & Consent Requirements and applicable privacy policy requirements as described in the User Data policy.
> Starting from July 11, 2021 midnight (UTC), new app releases containing this SDK version that do not meet the Prominent Disclosure & Consent Requirements per the User Data policy will be rejected. You may consider upgrading to 16.9.0, which the SDK provider has recommended for use instead. Please consult the SDK provider for further information."
> 
> Currently, I am using the Version 1.0.11 on Flutter 1.22.6, I can't use the latest version because it's using Dart 2.12 which uses flutter 2.
> Any workaround to how to overcome this issue ?

Did you find any solution?

**Comment by EjazAmir** [comment]
no, im still looking for solution?
if you find any, please do share

**Comment by EjazAmir** [comment]
> > "Your app ---- version code --- includes SDK com.stripe:stripe-android, 10.4.6 which facilitates the transmission or collection of personal and sensitive information. Such behavior must comply with Google Play's Prominent Disclosure & Consent Requirements and applicable privacy policy requirements as described in the User Data policy.
> > Starting from July 11, 2021 midnight (UTC), new app releases containing this SDK version that do not meet the Prominent Disclosure & Consent Requirements per the User Data policy will be rejected. You may consider upgrading to 16.9.0, which the SDK provider has recommended for use instead. Please consult the SDK provider for further information."
> > Currently, I am using the Version 1.0.11 on Flutter 1.22.6, I can't use the latest version because it's using Dart 2.12 which uses flutter 2.
> > Any workaround to how to overcome this issue ?
> 
> Did you find any solution?

Yes i found the solution, Stripe fixed the Google Compliance issue in hotFix 10.5.0 version. So now no need to upgrade to 16.x.x. you just need to fork the project version 1.0.11 and go to gradle.build file and just change the version from 10.4.6 to 10.5.0. i did and it works

**Comment by soarHero** [comment]
> "Your app ---- version code --- includes SDK com.stripe:stripe-android, 10.4.6 which facilitates the transmission or collection of personal and sensitive information. Such behavior must comply with Google Play's Prominent Disclosure & Consent Requirements and applicable privacy policy requirements as described in the User Data policy. Starting from July 11, 2021 midnight (UTC), new app releases containing this SDK version that do not meet the Prominent Disclosure & Consent Requirements per the User Data policy will be rejected. You may consider upgrading to 16.9.0, which the SDK provider has recommended for use instead. Please consult the SDK provider for further information."
> 
> Currently, I am using the Version 1.0.11 on Flutter 1.22.6, I can't use the latest version because it's using Dart 2.12 which uses flutter 2. Any workaround to how to overcome this issue ?

You can try 
```
stripe_payment:
    git:
      url: https://github.com/soarHero/flutter_stripe_payment.git
      ref: 52fff88d8986cd6c6a8fe3ec70472fdd8c511bd9
```
I removed the "null safety" code and can be used for Flutter 1.22.

**Comment by fabioselau077** [comment]
> 

Hello, thanks for share.
I installed this package and now in review for Play Store.

**Comment by fabioselau077** [comment]
> > "Your app ---- version code --- includes SDK com.stripe:stripe-android, 10.4.6 which facilitates the transmission or collection of personal and sensitive information. Such behavior must comply with Google Play's Prominent Disclosure & Consent Requirements and applicable privacy policy requirements as described in the User Data policy. Starting from July 11, 2021 midnight (UTC), new app releases containing this SDK version that do not meet the Prominent Disclosure & Consent Requirements per the User Data policy will be rejected. You may consider upgrading to 16.9.0, which the SDK provider has recommended for use instead. Please consult the SDK provider for further information."
> > Currently, I am using the Version 1.0.11 on Flutter 1.22.6, I can't use the latest version because it's using Dart 2.12 which uses flutter 2. Any workaround to how to overcome this issue ?
> 
> You can try
> 
> ```
> stripe_payment:
>     git:
>       url: https://github.com/soarHero/flutter_stripe_payment.git
>       ref: 52fff88d8986cd6c6a8fe3ec70472fdd8c511bd9
> ```
> 
> I removed the "null safety" code and can be used for Flutter 1.22.

Hello, thanks for share.
I installed this package and now in review for Play Store.
